### PR TITLE
Finish Chapter 8

### PR DESCRIPTION
**Pull request by tuanna59**

### CODE REVIEW

**Comment by tuanna59** [comment]
image

**Comment by tuanna59** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  # Returns true if the user is logged in, false otherwise.
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng current_user.present? nhé

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -1,11 +1,28 @@
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
-    <%= link_to "sample app", root_path, id: "logo" %>
+    <%= link_to (t ".app_title"), root_path, id: "logo" %>
```
Comment (dothidiemthao) [inline app/views/layouts/_header.html.erb]: t(".app_title")

**Comment by dothidiemthao** [comment]
commented

**Comment by tuanna59** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**